tasks/src/automatic-speech-recognition/about.md

```diff
@@ -1,3 +1,23 @@
+## Use Cases
+
+### Virtual Speech Assistants
+
+Many end devices have embedded a virtual assistance as a better way to interact with end users. These assistances rely on ASR models to recognize different voice commands in order to perform specified tasks. For instance, you might ask your phone for dialing a phone number, asking for an answer or scheduling a meeting. 
+
+### Caption Generation
+
+A Caption Generation model can take the audio from sources such as live streamed videos or recorded videos to generate automatic captions. This can help with content accessibility for example: audiences who are watching in a non-native language can rely on captions to interpret the content. Furthermore, it helps with information retention.
```

Comment:
```suggestion
A caption generation model takes audio as input from sources to generate automatic captions through transcription, for live-streamed or recorded videos. This can help with content accessibility. For example, an audience watching a video that includes a non-native language, can rely on captions to interpret the content. It can also help with information retention.
```
Can you elaborate on how it might help with information retention? 🙂